specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/instancePoolUsages.json

```diff
@@ -0,0 +1,343 @@
+              "modelAsString": true
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the instance pool usage.",
+            "schema": {
+              "$ref": "#/definitions/InstancePoolUsage"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InstancePoolWrongUsageName - Request for an instance pool's usage has an unsupported usage name\n\n * 400 InstancePoolManagedInstanceInfoUnavailable - Information for managed instances inside this instance pool is not available\n\n * 404 InstancePoolNotFound - An instance pool cannot be found"
+          }
+        },
+        "x-ms-examples": {
+          "Gets the instance pool vCore usage.": {
+            "$ref": "./examples/GetInstancePoolUsagesVCoreUtilization.json"
+          },
+          "Gets the instance pool storage usage.": {
+            "$ref": "./examples/GetInstancePoolUsagesStorageUtilization.json"
+          },
+          "Gets the instance pool database usage.": {
+            "$ref": "./examples/GetInstancePoolUsagesDatabaseUtilization.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "InstancePoolUsageListResult": {
+      "description": "A list of instance pool usages.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/InstancePoolUsage"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "InstancePoolUsageProperties": {
+      "description": "Properties of an instance pool usage.",
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "description": "User-readable name of the metric.",
+          "type": "string",
+          "readOnly": true
+        },
+        "unit": {
+          "description": "Unit of the metric",
+          "enum": [
+            "VCores",
+            "Gigabytes"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "InstancePoolUsageUnit",
+            "modelAsString": true
+          }
+        },
+        "currentValue": {
+          "format": "int32",
+          "description": "Instance pool current value.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "currentLimit": {
+          "format": "int32",
+          "description": "Instance pool current limit.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "requestedLimit": {
```

Comment:
>requestedLimit [](start = 9, length = 14)

this is just "limit" according to the spec.. is this PR updated based on the usages api spec shared offline ?.. for ex: unit doesn't allow VCores etc..

Reply:
updated to limit.